core/server/worker/src/main/java/alluxio/worker/netty/NettyDataServer.java

```diff
@@ -165,13 +158,17 @@ private ServerBootstrap createBootstrapOfType(final ChannelType type) {
     // If number of worker threads is 0, Netty creates (#processors * 2) threads by default.
     final int workerThreadCount =
         Configuration.getInt(PropertyKey.WORKER_NETWORK_NETTY_WORKER_THREADS);
-    final EventLoopGroup bossGroup =
-        NettyUtils.createEventLoop(type, bossThreadCount, "data-server-boss-%d", false);
-    final EventLoopGroup workerGroup =
-        NettyUtils.createEventLoop(type, workerThreadCount, "data-server-worker-%d", false);
-
-    final Class<? extends ServerChannel> socketChannelClass =
-        NettyUtils.getServerChannelClass(type);
+    String dataServerEventLoopNamePrefix =
+        "data-server-" + ((mSocketAddress instanceof DomainSocketAddress) ? "domain-socket" :
+            "tcp-socket");
```

Comment:
We either expect threads of one kind or the other but never both, so is it important to distinguish?